src/datumaro/plugins/transforms.py

```diff
@@ -1425,24 +1440,268 @@ def _analyze_reports(self, report):
                 label_id = self._extractor.categories()[AnnotationType.label].find(label_name)[0]
                 self._add_attrs[(rep["item_id"], rep["subset"])].append((label_id, attr_name))
 
+            if rep["anomaly_type"] == "RedundanciesInCaption":
+                desc = [s for s in str.split(rep["description"], "'")]
+                attr_name, label_name = desc[1], desc[3]
+                self._unnecessary_char_captions.append((label_name, attr_name))
+
+            if rep["anomaly_type"] == "EmptyLabel":
+                label = rep["description"].split("'")[1]
+                self._empty_labels[(rep["item_id"], rep["subset"])].append(label)
+
+            if rep["anomaly_type"] == "EmptyCaption":
+                caption = rep["description"].split("'")[1]
+                self._empty_captions[(rep["item_id"], rep["subset"])].append(caption)
+
+            if rep["anomaly_type"] == "OutlierInCaption":
```

Comment:
Instead of hardcoding the class name 'OutlierInCaption' as a string, you can use OutlierInCaption.__name__ to reference the class name. This technique can help prevent typos and is applicable for other scenarios where class name comparisons are necessary.

Reply:
Fix in [6f23b70](https://github.com/openvinotoolkit/datumaro/pull/1513/commits/6f23b70092ba956e262c859dffb400f764bf2fdc). Thanks for letting me know this.